extensions/timers/package.json

```diff
@@ -108,21 +89,13 @@
       "title": "Timers Menu Bar",
       "description": "View, start, or stop existing countdown timers from the Menu Bar.",
       "mode": "menu-bar",
-      "interval": "10s"
+      "interval": "1s"
```

Comment:
Maybe an unfinished PR, accidentally opened?

From the [Raycast docs](https://developers.raycast.com/information/lifecycle/background-refresh#manifest):

> The minimum value is 10 seconds (10s)